packages/transformers/js/core/src/lib.rs

```diff
@@ -252,6 +252,7 @@ pub fn transform(config: Config) -> Result<TransformResult, std::io::Error> {
                 ),
               ));
 
+              let is_module = module.is_module();
```

Comment:
What does this do exactly? What is the definition of a "module" here? Anything with an import/export? We also compute this within `collect` itself already.

Reply:
> Anything with an import/export

Looks like yes https://github.com/swc-project/swc/blob/a18ffc107f779d2fb576d610af65315e460efbd1/crates/swc_ecma_parser/src/parser/mod.rs#L162